tests/unit/OrgPerson.spec.ts

```diff
@@ -230,39 +228,38 @@ describe('Org Person component', () => {
   })
 
   it('Remove button is enabled in edit mode', async () => {
-    const wrapper: Wrapper<OrgPerson> = createComponent(validOrgData, 0, 0, null)
+    const wrapper: Wrapper<OrgPerson> = createComponent(validOrgData, 0, null)
     expect(wrapper.find(removeButtonSelector).attributes('disabled')).toBeUndefined()
     wrapper.destroy()
   })
 
   it('Remove button is disabled in create mode', async () => {
-    const wrapper: Wrapper<OrgPerson> = createComponent(validOrgData, -1, 0, null)
+    const wrapper: Wrapper<OrgPerson> = createComponent(validOrgData, -1, null)
     expect(wrapper.find(removeButtonSelector).attributes('disabled')).toBeDefined()
     wrapper.destroy()
   })
 
   it('Clicking remove button emits event', async () => {
-    const wrapper: Wrapper<OrgPerson> = createComponent(validOrgData, 0, 0, null)
+    const wrapper: Wrapper<OrgPerson> = createComponent(validOrgData, 0, null)
     wrapper.find(removeButtonSelector).trigger('click')
     await Vue.nextTick()
     expect(getLastEvent(wrapper, removePersonEvent)).toBe(0)
     wrapper.destroy()
   })
 
   it('Clicking Done button emits event for add edit person/org', async () => {
-    const wrapper: Wrapper<OrgPerson> = createComponent(validOrgData, -1, 0, null)
-    expect((<HTMLInputElement>wrapper.find(orgNameSelector).element).value)
-      .toEqual(validOrgData['officer']['orgName'])
+    const wrapper: Wrapper<OrgPerson> = createComponent(validOrgData, -1, null)
+    expect((<HTMLInputElement>wrapper.find(orgNameSelector).element).value).toEqual(validOrgData.officer.orgName)
     await Vue.nextTick()
     expect(wrapper.find(doneButtonSelector).attributes('disabled')).toBeUndefined()
     wrapper.find(doneButtonSelector).trigger('click')
     await Vue.nextTick()
-    expect(getLastEvent(wrapper, addEditPersonEvent)).toStrictEqual(validOrgData)
+    expect(getLastEvent(wrapper, addEditPersonEvent).officer.orgName).toEqual(validOrgData.officer.orgName)
```

Comment:
I had to change this as the "new officer logic" assigned a random id!

Also on line 391 below.